tools/keycloak/sync/SyncFactory.cs

```diff
@@ -230,7 +225,7 @@ private async Task RemoveRolesFromPims(IEnumerable<KModel.RoleModel> roles, Stri
                 try
                 {
                     _logger.LogInformation($"Deleting keycloak role '{role.Name}'");
-                    await _keycloakManagementClient.DeleteAsync($"{_keycloakManagementClient.GetIntegrationEnvUri()}/roles/{role.Name}");
+                    await _keycloakManagementClient.DeleteAsync($"{_keycloakManagementClient.GetIntegrationEnvUri()}/roles/{Uri.EscapeDataString(role.Name)}");
```

Comment:
the api expects % encoding on spaces and / characters, this is the only .net method that does both.